src/dom-to-image.js

```diff
@@ -6,6 +6,14 @@
     var fontFaces = newFontFaces();
     var images = newImages();
 
+    // Default impl options
+    var defaultOptions = {
+        // Default placeholder is a 1x1 gray square
+        placeholder: "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAIAAACQd1PeAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsMAAA7DAcdvqGQAAAAMSURBVBhXY7h79y4ABTICmGnXPbMAAAAASUVORK5CYII=",
```

Comment:
Ok I will change this to not have a default and the default behaviour will be to fail with an error which is the preexisting behaviour. There is no easy way to work with the browsers default broken image placeholder because that is applied to an img element and not to an XHR request, so I think failing is the best solution and if the developer includes a placeholder they have to explicitly set that, thus they know what it will be and are expecting that behaviour.